docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -168,9 +169,9 @@ to slow the test down.
 t.setTestSpeed( factor )
 ```
 
-Parameter  | Type      | Description
----------- | --------- | -----------
-`factor`   | Number    | Specifies the test speed. Must be a number between `1` (the fastest) and `0.01` (the slowest).
+Parameter  | Type      | Description | Default
+---------- | --------- | ----------- | ---------
+`factor`   | Number    | Specifies the test speed. Must be a number between `1` (the fastest) and `0.01` (the slowest). | `1`
```

Comment:
Are you sure that we should add a default value here?
Here we don't describe a `speed` option for test run which has a default value.
Here we describe a `setTestSpeed(factor)` function. `factor` is not an optional argument how it can't have default value